tests/transform/test_gene_info.py

```diff
@@ -301,3 +328,63 @@ def test_transform_gene_info_should_fail(
                 adjusted_p_value_threshold=param_set["adjusted_p_value_threshold"],
                 protein_level_threshold=param_set["protein_level_threshold"],
             )
+
+    @pytest.mark.parametrize(
+        "unpiprot_input_files, expected_output_file",
+        pass_uniprot_test_data,
+        ids=pass_uniprot_test_ids,
+    )
+    def test_add_uniprot_id_to_gene_info_should_pass(
```

Comment:
I'm a little stuck on whether this belongs in this file as an integration test or whether this is considered a unit test -- let me think on it.

Reply:
Now that it is a `collapse_uniprot_df()` function, it feels more like a unit test. I'll move it out of here.